modules/govuk_jenkins/templates/jobs/deploy_emergency_banner.yaml.erb

```diff
@@ -25,7 +25,59 @@
           LINK=$(escapeRakeArg $LINK)
           LINK_TEXT=$(escapeRakeArg $LINK_TEXT)
 
+          echo "Deploying emergency banner"
           ssh deploy@$(govuk_node_list -c frontend --single-node) "cd /var/apps/static && govuk_setenv static bundle exec rake \"emergency_banner:deploy[$CAMPAIGN_CLASS,$HEADING,$SHORT_DESCRIPTION,$LINK,$LINK_TEXT]\""
+
+          echo "Clearing template cache.."
```

Comment:
I wonder if we should extract this part and the part below about clearing the CDN caches into separate Jenkins jobs which get triggered from this job. If we don't then it's likely people will still need to use the Fabric scripts for just clearing some caches unrelated to the emergency publishing drill and it would be very easy for the cache clearing code in the Fabric scripts to end up different to the code here. My suggestion would be to move it into separate Jenkins tasks and then get rid of the Fabric scripts related to caching completely (or at least deprecate them for the moment).

As it is, people seem to prefer using Jenkins over the Fabric scripts so this might be nice anyway, and it allows us to more easily see when the tasks have been run and why.